ingest/src/main/java/com/microsoft/azure/kusto/ingest/result/ValidationPolicy.java

```diff
@@ -0,0 +1,68 @@
+package com.microsoft.azure.kusto.ingest.result;
+
+import java.io.Serializable;
+
+public class ValidationPolicy implements Serializable {
+    private ValidationOptions validationOptions = ValidationOptions.DoNotValidate;
+    private ValidationImplications validationPolicyType = ValidationImplications.BestEffort;
+
+    public ValidationPolicy() {
+    }
+
+    public ValidationPolicy(ValidationOptions validationOptions, ValidationImplications validationPolicyType) {
+        this.validationOptions = validationOptions;
+        this.validationPolicyType = validationPolicyType;
+    }
+
+    public ValidationPolicy(ValidationPolicy other) {
+        this.validationOptions = other.validationOptions;
+        this.validationPolicyType = other.validationPolicyType;
+    }
+
+    public ValidationOptions getValidationOptions() {
+        return validationOptions;
+    }
+
+    public void setValidationOptions(ValidationOptions validationOptions) {
+        this.validationOptions = validationOptions;
+    }
+
+    public ValidationImplications getValidationPolicyType() {
+        return validationPolicyType;
+    }
+
+    public void setValidationPolicyType(ValidationImplications validationPolicyType) {
+        this.validationPolicyType = validationPolicyType;
+    }
+
+    public enum ValidationOptions {
+        DoNotValidate("DoNotValidate"),
```

Comment:
Java standard is enums in ALL CAPS. The value in the parens allow using the case our service expects.

Reply:
Weird that IDEA ignores that by default, thanks!